fft/src/KokkosFFT_Plans.hpp

```diff
@@ -271,6 +275,81 @@ class Plan {
   Plan& operator=(Plan&&) = delete;
   Plan(Plan&&)            = delete;
 
+  /// \brief Execute FFT on input and output Views with normalization
+  ///
+  /// \param in [in] Input data
+  /// \param out [out] Ouput data
+  template <typename InViewType2, typename OutViewType2>
+  void execute(
+      const InViewType2& in, const OutViewType2& out,
+      KokkosFFT::Normalization norm = KokkosFFT::Normalization::backward) {
```

Comment:
```suggestion
  void execute(
      const InViewType2& in, const OutViewType2& out,
      KokkosFFT::Normalization norm = KokkosFFT::Normalization::backward) const {
```

Could it make sense to make the function `const` ?